Cargo.toml

```diff
@@ -69,5 +69,5 @@ proptest-derive = "0.4"
 rand = { version = "0.8", default-features = false }
 ruint = { version = "1.11.1", default-features = false, features = ["alloc"] }
 ruint-macro = { version = "1", default-features = false }
-winnow = { version = "0.5.19", default-features = false, features = ["alloc"] }
+winnow = { version = "0.5.39", default-features = false, features = ["alloc"] }
```

Comment:
@DaniPopes I needed to change this to get my local build working again after https://github.com/alloy-rs/core/pull/507

Not sure why CI checkout the correct version.

Reply:
Because the lockfile is not checked in, you have to run cargo update locally. But yes this should've been bumped

Reply:
can you open a separate PR for this?